.github/workflows/test.yml

```diff
@@ -46,9 +46,7 @@ jobs:
             args[1]="1"
             export GOMAXPROCS=1
           fi
-          export K6_BROWSER_EXECUTABLE_PATH=/usr/bin/google-chrome
-          export K6_BROWSER_HEADLESS=true
-          go test "${args[@]}" -timeout 5m ./...
+          K6_BROWSER_EXECUTABLE_PATH=/usr/bin/google-chrome K6_BROWSER_HEADLESS=true go test "${args[@]}" -timeout 5m ./...
```

Comment:
I tried this change but there are tests that rely on an explicit `K6_BROWSER_HEADLESS=false` otherwise they run in headful mode e.g. [TestFrameNoPanicWithEmbeddedIFrame](https://github.com/grafana/xk6-browser/blob/3015c48d5e502561aa8fd9ebf0766b4824b7e04b/tests/frame_test.go#L72).

Reply:
Oh OK! :+1: